src/starlite_saqlalchemy/worker.py

```diff
@@ -138,25 +139,48 @@ class JobConfig:
     """
 
 
+class CronJob(saq.CronJob):
+    """Cron Job."""
```

Comment:
is this for anything?

Reply:
It's main purpose now is for for convenience.  (i.e. you can import all of the classes from worker instead of having to import from `saq` as well).  I thought I might need to add additional functionality there, but I'm not sure at this point.